example/data-migration-ecs/main.tf

```diff
@@ -0,0 +1,30 @@
+terraform {
```

Comment:
I may just not be up-to-date on the migration setup for our end users, but aren't we going to need to package these as release artifact for the release series where we're supporting dual writes?  

If so, shouldn't be in `tf-modules` and be a configurable module?